api/Service/IUserService.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using YonderfulApi.Models;
+
+namespace YonderfulApi.Service
+{
+    public interface IUserService
+    {
+        Task<IList<User>> GetUserList();
+        Task<User> GetUserById(int userId);
+        Task<User> PostUser(string firstName, string lastName, string email, string password);
```

Comment:
Instead of passing each property individually, you should use an `User ` object as a parameter. 
Then if a new property is added to the User, you don't need to change the method signature.